server/chat-commands/core.ts

```diff
@@ -1321,6 +1321,19 @@ export const commands: ChatCommands = {
 		});
 	},
 
+	showbattlesinusercard: 'hidebattlesfromtrainercard',
+	hidebattlesfromusercard: 'hidebattlesfromtrainercard',
+	showbattlesintrainercard: 'hidebattlesfromtrainercard',
+	hidebattlesfromtrainercard(target, room, user, connection, cmd) {
+		const shouldHide = cmd.includes('hide');
+		user.settings.hideBattlesFromTrainerCard = shouldHide;
```

Comment:
You need to call user.update() or whatever to propagate the setting via |updateuser| to the client